test/testenv/deployment.go

```diff
@@ -669,7 +673,9 @@ func (d *Deployment) DeployMultisiteClusterWithSearchHeadAndAppFramework(name st
 			LicenseMasterRef: corev1.ObjectReference{
 				Name: licenseMaster,
 			},
-			Defaults: defaults,
+			Defaults:                     defaults,
+			LivenessInitialDelaySeconds:  int32(delaySeconds),
+			ReadinessInitialDelaySeconds: int32(delaySeconds),
```

Comment:
Nit: By default, the liveness probe is always 290 seconds higher than the readiness probe time. But, if this settings work fine for the test, that is fine as well :-)